tests/testthat/test-scalar_in_linter.R

```diff
@@ -15,16 +15,16 @@ test_that("scalar_in_linter skips allowed usages", {
 })
 
 test_that("scalar_in_linter blocks simple disallowed usages", {
-  linter <- scalar_in_linter()
-  lint_in_msg <- rex::rex("Use == to match length-1 scalars, not %in%.")
-  lint_chin_msg <- rex::rex("Use == to match length-1 scalars, not %chin%.")
+  linter <- scalar_in_linter(in_operators = "%chin%")
```

Comment:
Can we add a test with a vector of operators passed as an argument? 
You can use some additional binary infix operators from `{data.table}`.

Reply:
Use a vector of in operators to the second test